.moon/toolchain.yml

```diff
@@ -0,0 +1,72 @@
+# https://moonrepo.dev/docs/config/toolchain
+$schema: 'https://moonrepo.dev/schemas/toolchain.json'
+
+# Extend and inherit an external configuration file. Must be a valid HTTPS URL or file system path.
+# extends: './shared/toolchain.yml'
+
+# Configures Node.js within the toolchain. moon manages its own version of Node.js
+# instead of relying on a version found on the host machine. This ensures deterministic
+# and reproducible builds across any machine.
+node:
+  # The version to use. Must be a semantic version that includes major, minor, and patch.
+  # We suggest using the latest active LTS version: https://nodejs.org/en/about/releases
+  version: '19.5.0'
+
+  # The package manager to use when managing dependencies.
+  # Accepts "npm" (default), "pnpm", or "yarn".
+  packageManager: 'yarn'
+
+  # The version of the package manager (above) to use.
+  yarn:
+    version: '1.22.19'
+
+  # Add `node.version` as a constraint in the root `package.json` `engines`.
+  addEnginesConstraint: true
+
+  # Use the `package.json` name as an alias for the respective moon project.
+  aliasPackageNames: 'name-and-scope'
+
+  # Dedupe dependencies after the lockfile has changed.
+  dedupeOnLockfileChange: true
+
+  # Version format to use when syncing dependencies within the project's `package.json`.
+  # dependencyVersionFormat: 'workspace'
+
+  # Infer and automatically create moon tasks from `package.json` scripts, per project.
+  # BEWARE: Tasks and scripts are not 1:1 in functionality, so please refer to the documentation.
+  inferTasksFromScripts: false
+
+  # Sync a project's `dependsOn` as dependencies within the project's `package.json`.
+  syncProjectWorkspaceDependencies: true
+
+  # Sync `node.version` to a 3rd-party version manager's config file.
+  # Accepts "nodenv" (.node-version), "nvm" (.nvmrc), or none.
+  syncVersionManagerConfig: 'nvm'
+
+# Configures how moon integrates with TypeScript.
+typescript:
+  # When `syncProjectReferences` is enabled and a dependent project reference
+  # *does not* have a `tsconfig.json`, automatically create one.
+  createMissingConfig: false
+
+  # Name of `tsconfig.json` file in each project root.
+  # projectConfigFileName: 'tsconfig.json'
+
+  # Name of `tsconfig.json` file in the workspace root.
+  # rootConfigFileName: 'tsconfig.json'
+
+  # Name of the config file in the workspace root that defines shared compiler
+  # options for all project reference based config files.
+  # rootOptionsConfigFileName: 'tsconfig.options.json'
```

Comment:
any reason these are commented out?

Reply:
it's what moon scaffolds us with. I feel like it's nice to have these comments. wdyt?

Reply:
These are the defaults anyways.